### PR TITLE
Generate CE Unit Addressed ICL Print Files

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
Want to generate individual addressed initial contact letters for CE Units

# What has changed
Added pack codes and mappings
Created unit test

# How to test?
Build and run with other repos and then run ATs

# Links
https://trello.com/c/GEPhT08E/787-ic031-i041-generate-ce-unit-addressed-icl-print-files-3

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[(IC03.1/I04.1) Generate CE UNIT Addressed ICL Print Files (3)](https://trello.com/c/GEPhT08E/787-ic031-i041-generate-ce-unit-addressed-icl-print-files-3)

File: test/unit_tests/app/test_print_file_builder.py [hunk 1]
```diff
@@ -92,6 +92,35 @@ def test_generate_print_row_valid_CE1_IC01(cleanup_test_files):
...
+        "batchId": "1",
+        "batchQuantity": 3,
+        "uac": "test_uac",
+        "caseRef": "test_caseref",
+        "addressLine1": "123 Fake Street",
+        "addressLine2": "Duffryn",
+        "townName": "Newport",
+        "postcode": "NPXXXX",
+        "packCode": "D_ICA_ICLR1",
+        "qid": "31000000001234",
```
Comment (ryangrundy7) [inline test/unit_tests/app/test_print_file_builder.py:108]: This really should be a qid of type `21` since it's a individual CE unit

Comment (LeoKHoward) [reply test/unit_tests/app/test_print_file_builder.py:108]: Ah sorry, just made it up. Will change now

File: app/constants.py [hunk 2]
```diff
@@ -61,6 +65,10 @@ class PackCode(Enum):
     D_CE1A_ICLCR1 = 'D_CE1A_ICLCR1'
     D_CE1A_ICLCR2B = 'D_CE1A_ICLCR2B'
 
+    # Individual addressed initial contact letters for CE Units
```
Comment (ryangrundy7) [inline app/constants.py:68]: Only a comment but I think its the pack code for individual ICLs for CE estab and unit cases

**Pull request merged**